### DISCUSSION TITLE
Bible App Opportunity

### DISCUSSION THREAD

**Discussion by natcom95** [eliranwong/UniqueBible#1231] [Q&A]
Hi there,

I work for a worldwide ministry. Our pastor is a prolific, best-selling author with over 100 hundred titles sold across the globe and currently translated into over 50 languages. He writes on varying Christian topics, with the aim of building up the body of Christ. 

We are currently embarking on a project to create a reference bible that will link scripture quotations within his books to the bible, thus making studying the bible alongside the books much easier and linking the themes in the book to their relevant verses .

We are looking for an experienced bible app developer to join hands with us on this project and offer their expertise and guidance.

If this is something that you can help us with, I would love to shed more light on who are and what we hope to do. Please do get in touch and let's chat. 

Many thanks,
Natalie

**Comment by eliranwong** [comment]
Hi Natalie,

  Thanks for your message.  May I know more?  What is your organisation's name?  Who is your pastor?

  I think linking bible references within the book is easy to be done with our bible app.

Blessings,
Eliran

**Comment by natcom95** [comment]


**Reply by eliranwong** [reply]
You may send me an email first, so we may arrange a time

**Reply by natcom95** [reply]
Hi Eliran, how do I get your email address ?

**Comment by eliranwong** [comment]
https://www.bibletools.app/contact

Please leave your email in this form, thank you

**Comment by natcom95** [comment]
Hi Eliran 
I sent a message through the contact page on that website. Just checking you received it ?

**Reply by eliranwong** [reply]
> Hi Eliran I sent a message through the contact page on that website. Just checking you received it ?
> […](#)

I tried to locate your email, but unfortunately I cannot find it.

**Comment by eliranwong** [comment]
Hi, let me check and reply you.  Thank you.

**Comment by natcom95** [comment]
Hi Eliran

Please can you contact me on 

n.commey at outlook.com

Thanks

**Comment by natcom95** [comment]
Hi Eliran 

Just wondering if that email address worked okay for you ?

**Reply by eliranwong** [reply]
Sent
